recipes/userspace-rcu/all/test_package/test_urcu_fork.c

```diff
@@ -0,0 +1,214 @@
+/*
```

Comment:
would be nice to reduce it, calling one function (e.g. `rcu_register_thread`) is enough to test we can compile and link against the library